arcade/text.py

```diff
@@ -580,17 +579,13 @@ def create_text_sprite(
     italic: bool = False,
     anchor_x: str = "left",
     multiline: bool = False,
-    texture_atlas: Optional[arcade.TextureAtlas] = None,
-) -> arcade.Sprite:
+    texture_atlas: Optional[arcade.TextureAtlas] = None):
     """
-    Creates a sprite containing text based off of :py:class:`~arcade.Text`.
-
-    Internally this creates a Text object and an empty texture. It then uses either the
-    provided texture atlas, or gets the default one, and draws the Text object into the
-    texture atlas.
+    Creates a texture containing text based off of :py:class:`~pyglet.text.Label`.
 
-    It then creates a sprite referencing the newly created texture, and positions it
-    accordingly, and that is final result that is returned from the function.
+    Internally this creates a pyglet.text.Label object and an empty texture. It then uses either the
```

Comment:
Use this instead:
```
:py:class:`~pyglet.text.label`
```

This will yield RST formatted text. Also, try to wrap this line to eighty characters to make it more human-readable.